### PR TITLE
More readme updates.

### PR DESCRIPTION
**Pull request by Bostwickenator**
More updates to the readme file. Work in process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -278,7 +274,7 @@ iWF allows you to start a workflow with a fixed cron schedule like below
 // or timed out, the workflow will be retried based on the retry policy. While the workflow is retrying, it won't
 // schedule its next run. If the next schedule is due while the workflow is running (or retrying), then it will skip
 that
-// schedule. Cron workflow will not stop until it is terminated or cancelled (by returning cadence.CanceledError).
+// schedule. Cron workflow will not stop until it is terminated or canceled (by returning cadence.CanceledError).
```
Comment (Bostwickenator) [inline README.md:277]: cadence.CanceledError seems incorrect here given the two backends.

**Comment by Bostwickenator** [comment]
I'll resolve the conflicts shortly.

File: README.md (hunk 1 above)
Comment (longquanzheng) [reply README.md:277]: Ahh right. I think we just remove this part. Returning cancel error is not supported in IWF. The only to cancel a workflow is call the stop workflow api ( the stopType is cancel by default)

**Comment by longquanzheng** [comment]
> I'll resolve the conflicts shortly.

Btw there is only one conflict. There is a resolve conflict button in this PR and you can use it to resolve it easily

**Pull request closed without merging**